cmd/root.go

```diff
@@ -2,12 +2,27 @@ package cmd
 
 import (
 	"fmt"
+	"os"
 
+	log "github.com/Sirupsen/logrus"
 	"github.com/cozy/cozy-stack/config"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 )
 
+// ConfigFilename is the default configuration filename that cozy
+// search for
+const ConfigFilename = ".cozy"
+
+// ConfigPaths is the list of directories used to search for a
+// configuration file
+var ConfigPaths = []string{
+	".cozy",
+	"/etc/cozy",
+	"$HOME",
+	".",
+}
```

Comment:
It looks like you have added a config file. It's a good idea, but we should document it somewhere. Maybe in a `cli.md`, with other informations about the command-line (like the server mode)